### ISSUE TITLE
Flipping a Vortex Tube Retains Its Temperature Based on Direction Instead of the Relative Side

### ISSUE THREAD

**Issue by SwissxCore** [TeamPneumatic/pnc-repressurized#850]
### Minecraft Version
1.16.5

### Forge Version
36.1.0

### Mod Version
1.16.5-2.12.5-190

### Describe your problem, including steps to reproduce it
After using a vortex tube, dismantling it and flipping it, the temperature was retained based on the "north, east, south, or west" instead of the side of the block relative to its own orientation; for example: If you heat the north side and cool the south side, then flip the block, the north side should now be the cool side, and the south should be heated, as the block was flipped, but instead the temperature is retained despite this flip.

Step 1: Heat/cool opposite sides of the vortex tube through regular use.
Step 2: Dismantle the block.
Step 3: Place the vortex tube back down but flipped, so the previous north, east, south, or west side is now facing south, west, north, or east respectively.

### Any other comments?
The cold (blue) side of the vortex tube always places facing away from the player no matter the orientation.
When I discovered this I had placed the vortex tube "hot=north, cool=south" first, and later reversed.
Rotating the vortex tube using a pneumatic wrench does not rotate the temperature with it, further confirming that the temperature is fixed to the compass and not the block faces.

**Comment by desht** [comment]
Vortex tubes don't retain temperature at all when dropped as an item.  They retain _pressure_ (at least when wrenched - when pickaxed they lose any stored air).  It is this pressure which causes the temperature to change when the the vortex tube is placed back down.

Vortex tubes always place with the hot side facing toward the player; that is intentional and I see no need to change that.

Rotating a vortex tube most certainly does rotate the temperature with it.  What you may be seeing is the vortex tube being connected to some higher capacity heat source/sink; as soon as the tube is rotated, heat transfers to/from the blocks the vortex is now connected to.  Since the vortex tube heat handlers have a rather low heat capacity by default, their temperature will change very rapidly.

tl;dr there are no problems in PNC regarding the Vortex Tube & heat, just some slightly non-obvious but correct behaviour.
